easybuild/easyconfigs/g/GPyTorch/GPyTorch-1.9.1-foss-2021a-CUDA-11.3.1.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'PythonBundle'
+
+name = 'GPyTorch'
+version = '1.9.1'
+versionsuffix = '-CUDA-%(cudaver)s'
+
+homepage = 'https://gpytorch.ai'
+description = "GPyTorch is a Gaussian process library implemented using PyTorch."
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('CUDA', '11.3.1', '', SYSTEM),
+    ('Python', '3.9.5'),
+    ('PyTorch', '1.12.1', versionsuffix),
+    ('scikit-learn', '0.24.2'),
+]
+
+download_dep_fail = True
```

Comment:
This can be removed, as it is default for `PythonBundle` (and _should_ be removed in order to make the CI pass :))